### PR TITLE
FFmpegOutputBuilder - Minor tweaks

### PR DESCRIPTION
**Pull request by stonio**

### CODE REVIEW

File: src/main/java/net/bramp/ffmpeg/builder/FFmpegOutputBuilder.java [hunk 1]
```diff
@@ -516,7 +517,7 @@ public EncodingOptions buildOptions() {
     if (targetSize > 0) {
       checkState(parent.inputs.size() == 1, "Target size does not support multiple inputs");
 
-      String firstInput = parent.inputs.get(0);
+      String firstInput = parent.inputs.iterator().next();
```
Comment (bramp) [inline src/main/java/net/bramp/ffmpeg/builder/FFmpegOutputBuilder.java]: Is this better than get(0)?

Comment (stonio) [inline src/main/java/net/bramp/ffmpeg/builder/FFmpegOutputBuilder.java]: If you later replace `List` by `Collection` the trick `.iterator().next()` still works

Comment (bramp) [inline src/main/java/net/bramp/ffmpeg/builder/FFmpegOutputBuilder.java]: Ok, sure, it just seems a little less readable. But I'm not hugely concerned either way.

**Comment by bramp** [comment]
Thanks, lots of good changes!

**Pull request merged**